src/main/java/net/mcreator/element/converter/fv18/BiomeFrozenTopLayerConverter.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.converter.fv18;
+
+import com.google.gson.JsonElement;
+import net.mcreator.element.GeneratableElement;
+import net.mcreator.element.converter.IConverter;
+import net.mcreator.element.converter.fv12.BiomeDefaultFeaturesConverter;
+import net.mcreator.element.types.Biome;
+import net.mcreator.workspace.Workspace;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class BiomeFrozenTopLayerConverter implements IConverter {
+	private static final Logger LOG = LogManager.getLogger(BiomeFrozenTopLayerConverter.class);
+
+	@Override
+	public GeneratableElement convert(Workspace workspace, GeneratableElement input, JsonElement jsonElementInput) {
+		Biome biome = (Biome) input;
+		try {

```

Comment:
No need for a try-catch block in this case as no JSON parsing is done.

Reply:
Oops, I forgot to remove it when I copied the code of another converter.